sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -457,6 +457,9 @@ def generate_container_sas(
         using this shared access signature.
     :keyword str encryption_scope:
         Specifies the encryption scope for a request made so that all write operations will be service encrypted.
+    :keyword str correlation_id:
+        The correlation id to correlate the storage audit logs with the audit logs used by the principal
+        generating and distributing the SAS. This can only be used when to generate sas with delegation key.
```

Comment:
Hmm I know you copied this from Datalake but I guess maybe we should fix the typo? `when generating sas with delegation key.`?